src/components/Code.tsx

```diff
@@ -207,7 +211,12 @@ function renderLowLightNode(node: lowlight.AST.Unist.Node) {
       ? 1
       : 0;
   if (node.type === 'text') {
-    return <Text>{(node as lowlight.AST.Text).value}</Text>;
+    return (
+      <Text
+        key={createKey(index.toString(), (node as lowlight.AST.Text).value)}>
```

Comment:
Is the value of lowlight.AST.Text not unique enough on its own? Why do you need index in there too?

I mention this because as Keith said using index as a key is a bit of a red flag that you don't have a good stable index. In a list that never ever changes order it doesn't matter that much, so this is mostly academic. I believe the fallback behavior likely uses the order in the tree (same as index).

Reply:
I thought it would fail on repeated lines of text, but I just tried and it seems to be able to handle that case. Just updated the code.